library/src/G4CMPUtils.cc

```diff
@@ -211,6 +212,13 @@ void G4CMP::FillHit(const G4Step* step, G4CMPElectrodeHit* hit) {
 // Generate cos(theta) law for diffuse reflection, ensuring that computed
 // vector is directed inward with respect to the surface normal.
 
+G4ThreeVector
+G4CMP::GetLambertianVector(const G4LatticePhysical* theLattice,
+			   const G4ThreeVector& surfNorm, G4int mode) {
+  const G4ThreeVector surfPoint = GetCurrentTrack()->GetPosition();
+  return GetLambertianVector(theLattice, surfNorm, mode, surfPoint);
```

Comment:
You could do this as a one-liner if you wanted to:
```
  return GetLambertianVector(theLattice, surfNorm, mode, GetCurrentTrack()->GetPosition());
```
But the way you have it would let you print the vector in the debugger, so it doesn't need to be changed.